app/gulp/js/questions.js

```diff
@@ -58,15 +58,17 @@ $('body.questions.show').ready(() => {
   }
 
   function answerQuestion() {
-    const arr = $('.question_class').data('answers');
-    const arrlength = $('.question_class').data('numanswers');
-    console.log(arrlength);
+    const results = $('.question_class').data('results');
+    const answers = $('.question_class').data('answers');
```

Comment:
:+1: for meaningful variable names; I should have thought about this in my previous review!